quickstart/configure-mayastor.md

```diff
@@ -95,47 +95,49 @@ Pool configuration and state information can also be obtained by using the [Maya
 
 ## Create Mayastor StorageClass\(s\)
 
-Mayastor dynamically provisions Persistent Volumes \(PV\) based on StorageClass definitions created by the user. Parameters of the definition are used to set the characteristics and behaviour of its associated PVs. For a detailed description of these parameters see [storage class parameter description](https://mayastor.gitbook.io/introduction/reference/storage-class-parameters). Most importantly StorageClass definition is used to control the level of data protection afforded to it \(that is, the number of synchronous data replicas which are maintained, for purposes of redundancy\). It is possible to create any number of StorageClass definitions, spanning all permitted parameter permutations.
+Mayastor dynamically provisions PersistentVolumes \(PVs\) based on StorageClass definitions created by the user. Parameters of the definition are used to set the characteristics and behaviour of its associated PVs. For a detailed description of these parameters see [storage class parameter description](https://mayastor.gitbook.io/introduction/reference/storage-class-parameters). Most importantly, StorageClass definition is used to control the level of data protection afforded to it \(that is, the number of synchronous data replicas which are maintained, for purposes of redundancy\). It is possible to create any number of StorageClass definitions, spanning all permitted parameter permutations.
 
 We illustrate this quickstart guide with two examples of possible use cases; one which offers no data redundancy \(i.e. a single data replica\), and another having three data replicas. You can modify these as required to match your own desired test cases, within the limitations of the cluster under test.
 
 {% tabs %}
 {% tab title="Command \(1 replica example\)" %}
 ```text
 cat <<EOF | kubectl create -f -
-kind: StorageClass
 apiVersion: storage.k8s.io/v1
+kind: StorageClass
 metadata:
   name: mayastor-1
 parameters:
-  repl: '1'
-  protocol: 'nvmf'
-  ioTimeout: '60'
-  local: 'true'
+  ioTimeout: "30"
+  protocol: nvmf
+  repl: "1"
 provisioner: io.openebs.csi-mayastor
-volumeBindingMode: WaitForFirstConsumer
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
```

Comment:
I suspect the preference for `WaitForFirstConsumer` was originally intended for the "local" configuration, to place the application, the target/nexus, and the replica on the same node. 
Obviously when n>1 you can't have all replicas on the same node, and if the application moves to a different node then the replica is local no more.
Also with 2.0 we don't need to run the application and target/nexus on the same node..
So unless there's something I'm missing (and I may be), I don't think we need to suggest `WaitForFirstConsumer` from 2.0 onwards, at least for the general use case?

Reply:
For mayastor 2.0.0, which won't have the nexus HA, shouldn't we stick with "WaitForFirstConsumer" ?

Reply:
What are we trying to achieve by suggesting "WaitForFirstConsumer" ?

Reply:
Ok. Below is my understanding.
Unless we have high availability for nexus, we won't be able to separate the nexus and application pod.
Since nexus HA is not enabled in 2.0.0, we cannot separate nexus and application pod in 2.0.0, and this is typically enforced by "WaitForFirstConsumer".

Reply:
I don't "lock" nexus and application by using "WaitForFirstConsumer". That was done using the "local" flag which has now been removed.
Maybe we should add back the implicitly "local" on the 2.0 until we have HA?
An alternative option which we've discussed previously was to pin the applications to io-engine nodes.
@GlennBullingham @blaisedias